docs/products/flink/index.rst

```diff
@@ -1,5 +1,5 @@
-Aiven for Apache Flink
-=======================
+Flink :badge:`beta,cls=badge-secondary text-black badge-pill`
```

Comment:
Would it still fit if it was "Apache Flink" ?

Reply:
Yep, seen that we used Apache Kafka, we should do the same for Flink